assets/dev/js/editor/elements/views/container.js

```diff
@@ -34,6 +34,10 @@ const ContainerView = BaseElementView.extend( {
 		const behaviors = BaseElementView.prototype.behaviors.apply( this, arguments );
 
 		_.extend( behaviors, {
+			Sortable: {
```

Comment:
* TODO: Remove. It's a temporary solution for the Navigator sortable (also remove the behavior).